tests/e2e/sendEmail.test.ts

```diff
@@ -115,7 +120,7 @@ describe('web3mail.sendEmail()', () => {
         'Dataset order not found'
       );
     },
-    3 * MAX_EXPECTED_BLOCKTIME
+    2 * MAX_EXPECTED_BLOCKTIME + MAX_EXPECTED_WEB2_SERVICES_TIME
```

Comment:
this test includes a 5s sleep time
```suggestion
    2 * MAX_EXPECTED_BLOCKTIME + MAX_EXPECTED_WEB2_SERVICES_TIME + 5_000
```